### PR TITLE
Unpack rejection message before assertion in CommandHandlerExpected

### PR DESCRIPTION
**Pull request by vlad-lubenskyi**
This PR fixes an issue in `CommandHandlerExpected`. 

Previously, when calling `CommandHandlerExpected.throwsRejection()` the assertion was always made against `Rejection` instance. 

Now, the assertion is made against Rejection's `message`.

### CODE REVIEW

**Comment by vlad-lubenskyi** [comment]
@armiol PTAL

File: testutil-server/src/test/java/io/spine/testing/server/expected/CommandHandlerExpectedShould.java [hunk 1]
```diff
@@ -49,7 +49,7 @@
     void trackRejection() {
```
Comment (armiol) [inline testutil-server/src/test/java/io/spine/testing/server/expected/CommandHandlerExpectedShould.java]: We don't have `...Should`-named classes anymore. They all are called `...Test`. Please review the codebase you have recently created and update.

Comment (vlad-lubenskyi) [reply testutil-server/src/test/java/io/spine/testing/server/expected/CommandHandlerExpectedShould.java]: Should I leave `shoulds` in the cases where other wise I'd get `...TestTest`?

**Comment by vlad-lubenskyi** [comment]
@armiol PTAL

**Comment by vlad-lubenskyi** [comment]
@armiol PTAL

**Pull request merged**